src/Context/Context.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { createContext, useEffect, useState } from 'react';
+import { Person } from '../types';
+import { getPeople } from '../api';
+
+interface PeopleContextType {
+  people: Person[];
+  loader: boolean;
+  loadingError: boolean;
+  setPeople: (people: Person[]) => void;
+}
+
+export const PeopleContext = createContext<PeopleContextType>({
+  people: [],
+  loader: false,
+  loadingError: false,
+  setPeople: () => {},
```

Comment:
The `setPeople` function in the default value of `PeopleContext` is currently a no-op function. It would be better to provide a function that matches the expected type signature, even if it doesn't perform any operations, to avoid potential type errors.